assets/src/dashboard/app/views/editorSettings/adManagement/googleAdManager/test/googleAdManager.js

```diff
@@ -42,7 +42,7 @@ describe('Editor Settings: Google Analytics <GoogleAdManager />', function () {
   it('should render a visually hidden label for Google Ad Manager input', function () {
     const { getByLabelText } = renderWithProviders(
       <GoogleAdManagerSettings
-        adManagerSlotId={adManagerSlotId}
+        slotId={adManagerSlotId}
```

Comment:
What is this change?

Reply:
A bugfix to address prop type warnings.

The `GoogleAdManagerSettings` component expects a `slotId` prop, not `adManagerSlotId`.